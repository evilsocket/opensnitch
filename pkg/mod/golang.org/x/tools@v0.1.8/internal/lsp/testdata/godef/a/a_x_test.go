package a_test

import (
	"testing"
)

func TestA2(t *testing.T) { //@TestA2,godef(TestA2, TestA2)
	Nonexistant() //@diag("Nonexistant", "compiler", "undeclared name: Nonexistant", "error")
}
