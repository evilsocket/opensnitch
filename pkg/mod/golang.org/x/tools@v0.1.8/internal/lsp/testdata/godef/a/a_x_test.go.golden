-- TestA2-definition --
godef/a/a_x_test.go:7:6-12: defined here as ```go
func TestA2(t *testing.T)
```
-- TestA2-definition-json --
{
	"span": {
		"uri": "file://godef/a/a_x_test.go",
		"start": {
			"line": 7,
			"column": 6,
			"offset": 44
		},
		"end": {
			"line": 7,
			"column": 12,
			"offset": 50
		}
	},
	"description": "```go\nfunc TestA2(t *testing.T)\n```"
}

-- TestA2-hoverdef --
```go
func TestA2(t *testing.T)
```
